src/lib/__test__/use.px.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Theme } from 'theme-ui'
+import { themeToPx } from '../use-px'
+
+describe('themeToPx', () => {
+  it('Should work if sizes is an array', () => {
+    const theme: Theme = {
+      sizes: [0, 4, 8],
+    }
+
+    const k = 1
+    const result = themeToPx(theme, k, 'sizes')
+    expect(result).toBe(theme.sizes![k])
+  })
+
+  it('Should work if sizes is a smaller array ', () => {
```

Comment:
what does that mean? maybe, `should fallback to pixels if the value is greater than array length` or something like that